### PR TITLE
docs: fix docs and sample config

### PR DESCRIPTION
**Pull request by weizou19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,21 @@ Follow the steps to use NXLog for forwarding your Windows logs to LogDNA:
 ### Install NXLog
 
 * Install `NXLog Community Edition` from [here](https://nxlog.co/system/files/products/files/348/nxlog-ce-2.10.2150.msi), or
-* Run `choco install -y nxlog` on `PowerShell`
+* Run `choco install -y nxlog` on `PowerShell` (make sure choco has been installed before running this command)
```
Comment (smusali) [inline README.md]:
```suggestion
* Run `choco install -y nxlog` on `PowerShell` (make sure `choco` has been installed before running this command)
```

File: README.md [hunk 2]
```diff
@@ -37,11 +37,6 @@ $output = "$NXLOGDIR\cert\ca.pem"
 
 (New-Object System.Net.WebClient).DownloadFile($url, $output)
 ```
-
-### Modify LogDNA NXLog Configuration
```
Comment (smusali) [inline README.md:41]: Why did you remove this part?

**Review by smusali — CHANGES_REQUESTED** [review]
1 question, 1 suggestion

File: README.md (hunk 2 above)
Comment (weizou19) [reply README.md:41]: This is duplicate to this: https://github.com/logdna/logdna-nxlog/blob/master/README.md#copy-logdna-nxlog-configuration

**Review by smusali — APPROVED** [review]
